### ISSUE TITLE
Using PINCE on WSL

### ISSUE THREAD

**Issue by mljli** [korcankaraokcu/PINCE#103]
I know this may be a bit off-topic since WSL is not officially supported. But let me give the story...

I'm playing with PINCE in WSL and found that WSL does not support proc mem files. libscanmem fallbacked on ptrace() and errored out with message "no such process". By reading the man page of ptrace, I guess it's caused by libscanmem trying to attach the target process while it's already being attached to PINCE.

> ESRCH  The specified process does not exist, or is not currently being traced by the caller, or is not stopped (for requests that require a stopped tracee).

So, the question is can we make libscanmem to share the tracee with PINCE? I tried to comment out the PTRACE_ATTACH call in scanmem but this time PINCE was killed by signal 11 when I initiate a first scan.

Also, there's no popup for mem file not found. You can only find the error in the terminal.

**Comment by korcankaraokcu** [comment]
> So, the question is can we make libscanmem to share the tracee with PINCE?

No, ptrace disallows such behaviour. I don't think that we'll be redesigning the entire codebase to support WSL soon. PINCE also relies on proc mem for some of its functionality. So, WSL isn't and probably won't be supported in the near future, sorry about this.

**Comment by mljli** [comment]
Thanks for your quick response. No worries, I was not asking for support for WSL. Just curious about whether it could be solved.
